source/documentation/administration_guide/appe-enabling_FIPS.adoc

```diff
@@ -29,22 +29,18 @@ FIPS mode is enabled.
 ////
 [IMPORTANT]
 ====
-Red Hat recommends installing Red Hat Enterprise Linux 8 with FIPS mode enabled as opposed to enabling FIPS mode later. Enabling FIPS mode during the installation ensures that the system generates all keys with FIPS-approved algorithms and continuous monitoring tests in place.
+The FIPS mode needs to be enabled before RHV host deploy for Clusters with FIPS enabled. Enabling FIPS mode during the installation ensures that the system generates all keys with FIPS-approved algorithms and continuous monitoring tests in place.
```

Comment:
FIPS need to be enabled on engine too, otherwise the certificates are wrong. So the original sentence is better (with {enterprise-linux-shortname} instead, perhaps)

Reply:
Done